storage/pebble/config.go

```diff
@@ -46,19 +45,15 @@ func DefaultPebbleOptions(logger zerolog.Logger, cache *pebble.Cache, comparer *
 		l.FilterPolicy = bloom.FilterPolicy(MinLookupKeyLen)
 		l.FilterType = pebble.TableFilter
 
-		if i > 0 {
-			// L0 starts at 2MiB, each level is 2x the previous.
-			l.TargetFileSize = opts.Levels[i-1].TargetFileSize * 2
-		}
```

Comment:
option moved, but default matches our config anyway so I removed it:
https://github.com/RaduBerinde/pebble/blob/e0404d4b319d21f7d340bc27173e18020af04ada/options.go#L876-L877